src/sentry/models/groupredirect.py

```diff
@@ -11,6 +11,7 @@ class GroupRedirect(Model):
     """
     __core__ = False
 
+    orgnaization_id = BoundedBigIntegerField(null=True)
```

Comment:
we can backfill this one fairly easily given we already have group_id - ill follow up with that if we agree on this/push it out